src/mappings/Controller/mapping.ts

```diff
@@ -268,7 +268,7 @@ export function setGPParams(gpAddress: Address,
                             gpParamsHash: Bytes): void {
     let gp = GenesisProtocol.bind(gpAddress);
     let gpParams = GenesisProtocolParam.load(gpParamsHash.toHex());
-    if (gpParams == null) {
+    if (gpParams == null && !equalsBytes(gpParamsHash, new Bytes(32))) {
```

Comment:
yes, there can be a case where someone register a scheme with gp param as 0.by mistake or purpose .